internal/rbd/clone.go

```diff
@@ -188,6 +189,12 @@ func (rv *rbdVolume) doSnapClone(ctx context.Context, parentVol *rbdVolume) erro
 	if err != nil {
 		return err
 	}
+	err = tempClone.unsetAllMetadata(k8s.GetVolumeMetadataKeys())
```

Comment:
we should unset metadata at line 215 as if the `unsetAllMetadata` fails the tempClone need to be removed.

Reply:
ok